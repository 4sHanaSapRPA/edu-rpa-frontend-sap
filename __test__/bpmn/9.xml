<?xml version="1.0" encoding="UTF-8"?>
<bpmn:definitions xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:bpmn="http://www.omg.org/spec/BPMN/20100524/MODEL" xmlns:bpmndi="http://www.omg.org/spec/BPMN/20100524/DI" xmlns:dc="http://www.omg.org/spec/DD/20100524/DC" xmlns:di="http://www.omg.org/spec/DD/20100524/DI" id="Definitions_0jl7jys" targetNamespace="http://bpmn.io/schema/bpmn" exporter="bpmn-js (https://demo.bpmn.io)" exporterVersion="14.0.0">
  <bpmn:process id="Process_04og7m0" isExecutable="false">
    <bpmn:endEvent id="Event_10jy91v">
      <bpmn:incoming>Flow_07snrv8</bpmn:incoming>
    </bpmn:endEvent>
    <bpmn:startEvent id="StartEvent_1mxxylx">
      <bpmn:outgoing>Flow_0x254yi</bpmn:outgoing>
    </bpmn:startEvent>
    <bpmn:subProcess id="Activity_1pjcvad">
      <bpmn:incoming>Flow_0x254yi</bpmn:incoming>
      <bpmn:outgoing>Flow_07snrv8</bpmn:outgoing>
      <bpmn:startEvent id="Event_07li38w">
        <bpmn:outgoing>Flow_1kiz9g2</bpmn:outgoing>
      </bpmn:startEvent>
      <bpmn:task id="Activity_0pkjxtm" name="Open Browser">
        <bpmn:incoming>Flow_1kiz9g2</bpmn:incoming>
        <bpmn:outgoing>Flow_1bge0c6</bpmn:outgoing>
      </bpmn:task>
      <bpmn:task id="Activity_1536xo5" name="Get Title Assign to ${title}">
        <bpmn:incoming>Flow_1bge0c6</bpmn:incoming>
        <bpmn:outgoing>Flow_0g1mo6a</bpmn:outgoing>
      </bpmn:task>
      <bpmn:task id="Activity_012ckfs" name="Close Browser">
        <bpmn:incoming>Flow_0g1mo6a</bpmn:incoming>
        <bpmn:outgoing>Flow_1e9rifc</bpmn:outgoing>
      </bpmn:task>
      <bpmn:endEvent id="Event_1mg11tg">
        <bpmn:incoming>Flow_1e9rifc</bpmn:incoming>
      </bpmn:endEvent>
      <bpmn:sequenceFlow id="Flow_1kiz9g2" sourceRef="Event_07li38w" targetRef="Activity_0pkjxtm" />
      <bpmn:sequenceFlow id="Flow_1bge0c6" sourceRef="Activity_0pkjxtm" targetRef="Activity_1536xo5" />
      <bpmn:sequenceFlow id="Flow_0g1mo6a" sourceRef="Activity_1536xo5" targetRef="Activity_012ckfs" />
      <bpmn:sequenceFlow id="Flow_1e9rifc" sourceRef="Activity_012ckfs" targetRef="Event_1mg11tg" />
    </bpmn:subProcess>
    <bpmn:sequenceFlow id="Flow_0x254yi" sourceRef="StartEvent_1mxxylx" targetRef="Activity_1pjcvad" />
    <bpmn:sequenceFlow id="Flow_07snrv8" sourceRef="Activity_1pjcvad" targetRef="Event_10jy91v" />
  </bpmn:process>
  <bpmndi:BPMNDiagram id="BPMNDiagram_1">
    <bpmndi:BPMNPlane id="BPMNPlane_1" bpmnElement="Process_04og7m0">
      <bpmndi:BPMNShape id="_BPMNShape_StartEvent_2" bpmnElement="StartEvent_1mxxylx">
        <dc:Bounds x="152" y="162" width="36" height="36" />
      </bpmndi:BPMNShape>
      <bpmndi:BPMNShape id="Event_10jy91v_di" bpmnElement="Event_10jy91v">
        <dc:Bounds x="1062" y="162" width="36" height="36" />
      </bpmndi:BPMNShape>
      <bpmndi:BPMNShape id="Activity_1pjcvad_di" bpmnElement="Activity_1pjcvad" isExpanded="true">
        <dc:Bounds x="250" y="60" width="700" height="270" />
      </bpmndi:BPMNShape>
      <bpmndi:BPMNShape id="Event_07li38w_di" bpmnElement="Event_07li38w">
        <dc:Bounds x="292" y="172" width="36" height="36" />
      </bpmndi:BPMNShape>
      <bpmndi:BPMNShape id="BPMNShape_1fopg3k" bpmnElement="Event_1mg11tg">
        <dc:Bounds x="862" y="172" width="36" height="36" />
      </bpmndi:BPMNShape>
      <bpmndi:BPMNShape id="Activity_0pkjxtm_di" bpmnElement="Activity_0pkjxtm">
        <dc:Bounds x="380" y="150" width="100" height="80" />
        <bpmndi:BPMNLabel />
      </bpmndi:BPMNShape>
      <bpmndi:BPMNShape id="Activity_1536xo5_di" bpmnElement="Activity_1536xo5">
        <dc:Bounds x="540" y="150" width="100" height="80" />
        <bpmndi:BPMNLabel />
      </bpmndi:BPMNShape>
      <bpmndi:BPMNShape id="BPMNShape_0eltz4d" bpmnElement="Activity_012ckfs">
        <dc:Bounds x="700" y="150" width="100" height="80" />
        <bpmndi:BPMNLabel />
      </bpmndi:BPMNShape>
      <bpmndi:BPMNEdge id="Flow_1kiz9g2_di" bpmnElement="Flow_1kiz9g2">
        <di:waypoint x="328" y="190" />
        <di:waypoint x="380" y="190" />
      </bpmndi:BPMNEdge>
      <bpmndi:BPMNEdge id="Flow_1bge0c6_di" bpmnElement="Flow_1bge0c6">
        <di:waypoint x="480" y="190" />
        <di:waypoint x="540" y="190" />
      </bpmndi:BPMNEdge>
      <bpmndi:BPMNEdge id="Flow_0g1mo6a_di" bpmnElement="Flow_0g1mo6a">
        <di:waypoint x="640" y="190" />
        <di:waypoint x="700" y="190" />
      </bpmndi:BPMNEdge>
      <bpmndi:BPMNEdge id="Flow_1e9rifc_di" bpmnElement="Flow_1e9rifc">
        <di:waypoint x="800" y="190" />
        <di:waypoint x="862" y="190" />
      </bpmndi:BPMNEdge>
      <bpmndi:BPMNEdge id="Flow_0x254yi_di" bpmnElement="Flow_0x254yi">
        <di:waypoint x="188" y="180" />
        <di:waypoint x="250" y="180" />
      </bpmndi:BPMNEdge>
      <bpmndi:BPMNEdge id="Flow_07snrv8_di" bpmnElement="Flow_07snrv8">
        <di:waypoint x="950" y="180" />
        <di:waypoint x="1062" y="180" />
      </bpmndi:BPMNEdge>
    </bpmndi:BPMNPlane>
  </bpmndi:BPMNDiagram>
</bpmn:definitions>
