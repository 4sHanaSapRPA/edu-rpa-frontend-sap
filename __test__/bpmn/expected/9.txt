BlankBlock: Activity_1pjcvad
	Activity_0pkjxtm - Open Browser
	Activity_1536xo5 - Get Title Assign to ${title}
	Activity_012ckfs - Close Browser